pkg/lib/form-layout.less

```diff
@@ -223,9 +239,8 @@
       // Don't restrict grid placement
       grid-column: auto;
       // Use default PatternFly line height
-      line-height: inherit;
+      line-height: @line-height-base;
```

Comment:
The variables in this scope come from PatternFly, not the local definitions above (which only apply to everything nested in `.ct-form-layout`).

Should I make that clear in a comment?